frontend/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            URL Shortener
+        </title>
+        <script src="script.js"></script>
```

Comment:
Since you don't need it, no need to import.

Reply:
yeah